cmd/cassowary/cli.go

```diff
@@ -176,10 +176,26 @@ func validateCLI(c *cli.Context) error {
 		httpMethod = "GET"
 	}
 
-	tlsConfig := new(tls.Config)
+    tlsConfig := new(tls.Config)
 	if c.Bool("insecure") {
 		tlsConfig.InsecureSkipVerify = true
-	}
+    }
+
+    if c.String("renegotiation") != "" {
+        switch option := c.String("renegotiation"); option {
+        case "never":
+            tlsConfig.Renegotiation = tls.RenegotiateNever
+        case "once":
+            tlsConfig.Renegotiation = tls.RenegotiateOnceAsClient
+        case "freely":
+            tlsConfig.Renegotiation = tls.RenegotiateFreelyAsClient
+        default:
+            return fmt.Errorf("invalid renegotiation option specified: %q", option)
+        }
+	} else {
+        tlsConfig.Renegotiation = tls.RenegotiateNever
+    }
```

Comment:
Skip the else since this will be the default value anyways:

```go
    if c.String("renegotiation") != "" {
        switch option := c.String("renegotiation"); option {
        case "never":
            tlsConfig.Renegotiation = tls.RenegotiateNever
        case "once":
            tlsConfig.Renegotiation = tls.RenegotiateOnceAsClient
        case "freely":
            tlsConfig.Renegotiation = tls.RenegotiateFreelyAsClient
        default:
            return fmt.Errorf("invalid renegotiation option specified: %q", option)
        }
    }
```